exotic.py

```diff
@@ -523,6 +540,25 @@ def plate_solution(fits_file, saveDirectory):
             return False
         time.sleep(5)
 
+#Checks if comparison star coordinates don't point to variable stars
+def variableStarCheck(comparisonStarFile):
+    #Open WCS file and convert comparison star's coordinates into normal coordinates
+    starList = fits.open(comparisonStarFile)
+    wcsCoords = WCS.WCS(starList[0].header)
+    starCoords = wcsCoords.all_pix2world()
+
+    #Use normalized Coordinates to search on SIMBAD to check if that star is variable
+    try:
+        resultTable = Simbad.query_region(coord.SkyCoord(starCoords[0], starCoords[1], unit = (u.deg, u.deg))
+    except TypeError:
+        print("Error: Invalid star coordinates. Aborting process")
```

Comment:
If the user gets to this line of the code (i.e., they get a `TypeError`, resultTable variable will not be initialized, and they will get an error at line 557. Suggest just putting in a `return False` line here after this print statement or do something else.

Reply:
Please see updated code